api/app/presenters/spree/products/option_type_filters_presenter.rb

```diff
@@ -0,0 +1,39 @@
+module Spree
```

Comment:
@rafalcymerys why not re-use https://github.com/spree/spree/blob/main/core/app/presenters/spree/filters/options_presenter.rb ?

Reply:
That was my dilemma here ;) Theoretically we could, but I'm wondering whether it's a good idea to couple the presentation the layer of the `api` module with some hashes returned by the`core` module. We'd need to add to_h method to the presenter anyway.

My intuition was that since that's API-specific implementation, it would make sense to have the presenter in the API. Any thoughts here?

Reply:
@rafalcymerys we could, API is the core :) there won't be any other presentations ;)